### PR TITLE
Add support Percona tools 8.0 repo.

### PR DESCRIPTION
**Pull request by MemberIT**
Add support Percona tools 8.0 repo.
Deps: https://github.com/Oefenweb/ansible-percona-server/pull/33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ None
 
 #### Variables
 
+##### Percona version
+
+* `percona_toolkit_repo_version`: [required, default: `5.6`]: Manages the repository depending on the Percona version (5.5, 5.6, 5.7 and 8.0)
```
Comment (tersmitten) [inline README.md:15]: To be consistent with the other roles I think I prefer `percona_toolkit_version`, but it's not really the toolkit version...

File: vars/main.yml [hunk 2]
```diff
@@ -1,12 +1,28 @@
...
 
+percona_toolkit_repositories_8:
+  - type: deb
+    url: "http://repo.percona.com/tools/apt {{ ansible_lsb.codename }}"
+    component: main
+  - type: deb-src
+    url: "http://repo.percona.com/tools/apt {{ ansible_lsb.codename }}"
+    component: main
+
+percona_toolkit_repositories: "{{ percona_toolkit_repositories_8 if percona_toolkit_repo_version == 8.0 else percona_toolkit_repositories_5 }}"
```
Comment (tersmitten) [inline vars/main.yml]: Let's use the `version` filter here

File: README.md (hunk 1 above)
Comment (MemberIT) [reply README.md:15]: I thought so too, but it seems to me that this variable name `percona_toolkit_repo_version` is more correct.

**Comment by tersmitten** [comment]
Apparently there is no longer need for this PR. `master` is already fine.

**Pull request closed without merging**